cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -190,7 +190,10 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
             inputDir, distinctPackageManagers, curationProvider, repositoryConfiguration
         ).mergeLabels(labels)
 
-        println("Found ${ortResult.getProjects().size} project(s) in total.")
+        val analyzedProjects = ortResult.getProjects()
+        val countPerType = analyzedProjects.groupingBy { it.id.type }.eachCount()
+        val projectCountDetails = countPerType.toSortedMap().map { (type, count) -> "$type ($count)" }.joinToString()
```

Comment:
Would it make sense to pass the lambda `{ (type, count) -> "$type ($count)" }` to `joinToString()` instead of to `map()`?

Reply:
Unfortunately, I cannot call `joinToString()` on a map directly, so

    countPerType.toSortedMap().joinToString { (type, count) -> "$type ($count)" }

does not work. However, I could do

    countPerType.toSortedMap().entries.joinToString { (type, count) -> "$type ($count)" }

Not sure if that's nicer, though.

Reply:
Yeah, I kind of assumed already you'd add the `\n` as well, like:

`countPerType.toSortedMap().entries.joinToString(prefix = "  ", separator="\n  ") { (type, count) -> "$type ($count)" }`

I'll leave it up to you to decide, no strong opinion here.